db/seeds.rb

```diff
@@ -1,9 +1,18 @@
-# db/seeds.rb
-
 require "faker"
 
+# Don't send emails during seed process.
+ActionMailer::Base.perform_deliveries = false
+
+# Clear existing data
+ActiveRecord::Base.connection.execute("TRUNCATE logbook_entries, aircrafts, users, notifications RESTART IDENTITY CASCADE")
+
+USERS_COUNT = 3
```

Comment:
```
USERS_COUNT = 10
AIRCRAFT_NOTIFICATIONS_COUNT = 1000
LOGBOOK_ENTRY_NOTIFICATIONS = 1000
USER_NOTIFICATIONS = 100
```

shouldn't push the test values to the repo (we'd just need to make that change locally to test and then discard the change)
- but after confirming that the seeding works with smaller numbers, we'd want our constants to be 10, 1000, etc for when we seed the DB in preparation for the upgrade

Reply:
Sure, the change is done, Thx